gpytorch/lazy/toeplitz_lazy_variable.py

```diff
@@ -2,17 +2,22 @@
 import torch
 from gpytorch.utils import toeplitz
 from .lazy_variable import LazyVariable
-from gpytorch.math.functions.lazy_toeplitz import InterpolatedToeplitzGPMarginalLogLikelihood
+from gpytorch.math.functions.lazy_toeplitz import InterpolatedToeplitzGPMarginalLogLikelihood, \
+    ToeplitzMVNKLDivergence
 
 
 class ToeplitzLazyVariable(LazyVariable):
-    def __init__(self, c, r, J_left=None, C_left=None, J_right=None, C_right=None, added_diag=None):
+    def __init__(self, c, r, inducing_points=None,
```

Comment:
I'm not a fan of inducing points living here. We should talk about another way of doing this.